src/control/server/ctl_storage_rpc_test.go

```diff
@@ -773,9 +756,33 @@ func TestStoragePrepare(t *testing.T) {
 //				t.Fatal(fmtErr)
 //			}
 //
-//			results := []*StorageFormatResp{resp}
-//			if diff := cmp.Diff(tc.expResults, results); diff != "" {
-//				t.Fatalf("unexpected results: (-want, +got):\n%s\n", diff)
+//			common.AssertEqual(t, len(tc.expResp.Crets), len(resp.Crets),
+//				"number of controller results")
+//			common.AssertEqual(t, len(tc.expResp.Mrets), len(resp.Mrets),
+//				"number of mount results")
+//			for _, exp := range tc.expResp.Crets {
+//				match := false
+//				for _, got := range resp.Crets {
+//					if diff := cmp.Diff(exp, got); diff == "" {
+//						match = true
+//					}
+//				}
+//				if !match {
+//					t.Fatalf("unexpected results: (\nwant: %+v\ngot: %+v)",
+//						tc.expResp.Crets, resp.Crets)
+//				}
+//			}
+//			for _, exp := range tc.expResp.Mrets {
+//				match := false
+//				for _, got := range resp.Mrets {
+//					if diff := cmp.Diff(exp, got); diff == "" {
+//						match = true
+//					}
+//				}
+//				if !match {
+//					t.Fatalf("unexpected results: (\nwant: %+v\ngot: %+v)",
+//						tc.expResp.Mrets, resp.Mrets)
+//				}
```

Comment:
Are all the tests being re-enabled in another PR?

Reply:
yes . after applying the new client API to fanout in https://github.com/daos-stack/daos/pull/1778